src/legacy/core_plugins/metrics/public/components/vis_types/table/vis.js

```diff
@@ -48,10 +49,9 @@ class TableVis extends Component {
     this.renderRow = this.renderRow.bind(this);
   }
 
-  renderRow(row) {
-    const { model } = this.props;
-    const rowId = row.key;
-    let rowDisplay = rowId;
+  renderRow = (isDateTypeField = false) => row => {
+    const { model, dateFormat } = this.props;
+    let rowDisplay = isDateTypeField ? moment(row.key).format(dateFormat) : row.key;
```

Comment:
Although this way to format dates is spread across metrics and other apps, I'm kindly ask you if you can try to use the `DateFormat` in `src/legacy/core_plugins/kibana/common/field_formats/types/date.js`. This will, in the future, helps us to migrate from `moment` to an immutable and lightweight date library

Reply:
Done. Please, check it